### PR TITLE
Image manifest unclear sentence

### PR DESCRIPTION
**Pull request by jonboulle**
"used as a human readable index to the App Container Image" doesn't mean anything to me.  Needs a better description.

### CODE REVIEW

**Comment by jonboulle** [comment]
@thockin is this sufficient?

File: SPEC.md [hunk 1]
```diff
@@ -732,7 +732,7 @@ JSON Schema for the Image Manifest (app image manifest, ACI manifest), conformin
 
 * **acKind** (string, required) must be an [AC Kind](#ac-kind-type) of value "ImageManifest"
 * **acVersion** (string, required) represents the version of the schema specification [AC Version Type](#ac-version-type)
-* **name** (string, required) used as a human readable index to the App Container Image. (string, restricted to the [AC Name](#ac-name-type) formatting)
+* **name** (string, required) a human-readable name for this App Container Image (string, restricted to the [AC Name](#ac-name-type) formatting). This is not expected to be unique (see the **version** label) but SHOULD have a URL-like structure to facilitate **[App Container Image Discovery](#app-container-image-discovery)**.
```
Comment (thockin) [inline SPEC.md]: Is it assumed that this name, when resolved through the discovery process, will result in this same image?  Might be worth saying something like that.

Comment (jonboulle) [inline SPEC.md]: We do describe that [here](https://github.com/appc/spec/blob/master/SPEC.md#validation):
> Implementations must ensure that the name in the Image Manifest in the retrieved ACI matches the initial name used for discovery.

This is one of those awkward moments where we have erred on the side of brevity and not duplicated information between the manifest schema and the body of the spec, but referencing it again here would arguably be useful too... any strong feelings?

Comment (thockin) [inline SPEC.md]: I tend to err on the side of being too explicit.  In this case I think it
is an important point.

**Comment by jonboulle** [comment]
PTAL

**Comment by thockin** [comment]
LGTM

**Pull request merged**